pkg/storaged/client.js

```diff
@@ -365,6 +373,22 @@ function btrfs_start_monitor() {
     channel.catch(err => {
         throw new Error(err.toString());
     });
+
+    btrfs_monitor_channel = channel;
+}
+
+client.btrfs_stop_monitoring = () => {
+    if (btrfs_monitor_channel) {
+        console.log("STOPPING");
+        const res = btrfs_monitor_channel.then(() => {
+            console.log("(stopped)");
+            btrfs_monitor_channel = null;
+        });
+        btrfs_monitor_channel.close();
+        return res;
+    } else {
+        return Promise.resolve();
```

Comment:
This added line is not executed by any test.

Reply:
Very good, this means we never try to stop the monitor when it is not actually running.